.gitignore

```diff
@@ -108,4 +108,7 @@ yalc.lock
 .cxx/
 
 # Temporary files created by Metro to check the health of the file watcher
-.metro-health-check*
\ No newline at end of file
+.metro-health-check*
+
+# Test coverage reports
+/coverage
```

Comment:
I would like to see the introduction of coverage in a separate PR and if we can use an opensource tool like https://about.codecov.io/ and put a label in our repo.